lib/client.ts

```diff
@@ -0,0 +1,361 @@
+import { ServiceClientRequestOptions, ServiceClientResponse, request } from './request'
+import * as url from 'url'
+
+const retry = require('retry')
```

Comment:
Would be nice to have a note why these are `require` statements and not `import`

Reply:
would be even nicer to have them proper imports -_-'

I'll add a note